### ISSUE TITLE
Submit form request error -  frontend not validate

### ISSUE THREAD

**Issue by sapisuper** [kviklet/kviklet#240]
Hi,

I got this error when submitting request, cause no validation character in frontend side. please check further, for workaround I think we can change data type from varchar to text for column description.

thanks

```
2024-09-20T08:21:02.659Z DEBUG 8 --- [nio-8081-exec-9] org.hibernate.SQL                        : insert into execution_request (author_id,command,datasource_id,container_name,created_at,description,execution_request_type,execution_status,execution_type,namespace,pod_name,statement,title,id) values (?,?,?,?,?,?,?,?,?,?,?,?,?,?)
2024-09-20T08:21:02.661Z  WARN 8 --- [nio-8081-exec-9] o.h.engine.jdbc.spi.SqlExceptionHelper   : SQL Error: 0, SQLState: 22001
2024-09-20T08:21:02.661Z ERROR 8 --- [nio-8081-exec-9] o.h.engine.jdbc.spi.SqlExceptionHelper   : ERROR: value too long for type character varying(255)
2024-09-20T08:21:02.662Z ERROR 8 --- [nio-8081-exec-9] d.k.k.c.ExceptionHandlerController       : Exception occurred at /execution-requests/
```

**Comment by sapisuper** [comment]
Workaround: change data type for column description 

```
kviklet=# \d execution_request
                                       Table "public.execution_request"
         Column         |            Type             | Collation | Nullable |             Default             
------------------------+-----------------------------+-----------+----------+---------------------------------
 id                     | character(22)               |           | not null | 
 datasource_id          | character varying(255)      |           | not null | 
 author_id              | character varying(22)       |           | not null | 
 title                  | character varying(255)      |           | not null | 
 execution_type         | character varying(255)      |           | not null | 
 description            | character varying(255)      |           |          | 
 statement              | text                        |           |          | 
 execution_status       | character varying(255)      |           | not null | 
 created_at             | timestamp without time zone |           | not null | now()
 namespace              | character varying(255)      |           |          | 
 pod_name               | character varying(255)      |           |          | 
 container_name         | character varying(255)      |           |          | 
 command                | text                        |           |          | 
 execution_request_type | character varying(255)      |           | not null | 'DATASOURCE'::character varying
Indexes:
    "execution_request_pkey" PRIMARY KEY, btree (id)                                                         ^
kviklet=# 
kviklet=# ALTER TABLE execution_request ALTER COLUMN description TYPE text;
ALTER TABLE
kviklet=#

**Comment by Askir** [comment]
Fixed in #245 thanks for mentioning this! I assume there is more of these mismatches in the code base. Sometimes I iterate a bit fast and don't try all the edge cases 😅 thanks for reporting these!
